client/graphics/passes/debug-pass.hpp

```diff
@@ -10,13 +10,14 @@ namespace tec::graphics::pass {
 class DebugPass final : public RenderPass {
 public:
 	DebugPass() : RenderPass("DebugPass") {}
-	void Prepare(const GBuffer& gbuffer) override {
+	void Prepare(GBuffer& gbuffer) override {
 		gbuffer.BindForRendering();
 		glActiveTexture(GL_TEXTURE0 + static_cast<int>(GBuffer::DEPTH_TYPE::DEPTH));
 		glBindSampler(static_cast<int>(GBuffer::DEPTH_TYPE::DEPTH), 0);
 		glBindTexture(GL_TEXTURE_2D, gbuffer.GetDepthTexture());
 	}
-	void Run(const gfx::ShaderSet& default_shaders, const View&, const RenderItems&) override {
+	void
+	Run(const gfx::ShaderSet& default_shaders, const Viewport& viewport, const View&, const RenderItems&) override {
```

Comment:
are return types supposed to be on new lines? (I noticed this in three places)

Reply:
clang-format did that, but I'll give it a double check

Reply:
I renamed the parameter and it cleaned it up some